### PR TITLE
chore: Additional metadata needed for SLF4j 2 in native images

### PR DESCRIPTION
**Pull request by johanandren**
References #32503

### CODE REVIEW

**Comment by johanandren** [comment]
Green native image test run in https://github.com/akka/akka/actions/runs/10703817268

File: native-image-tests/local-scala/build.sbt [hunk 1]
```diff
@@ -28,6 +28,6 @@ libraryDependencies ++= Seq(
   "com.typesafe.akka" %% "akka-actor-typed" % akkaVersion,
   "com.typesafe.akka" %% "akka-stream" % akkaVersion,
   "com.typesafe.akka" %% "akka-serialization-jackson" % akkaVersion,
-  "ch.qos.logback" % "logback-classic" % "1.2.13",
+  "ch.qos.logback" % "logback-classic" % "1.5.7",
```
Comment (patriknw) [inline native-image-tests/local-scala/build.sbt:31]: searching for `1.2.13` I see many more in samples

Comment (patriknw) [reply native-image-tests/local-scala/build.sbt:31]: ok, covered by https://github.com/akka/akka/pull/32490

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**